README.md

```diff
@@ -49,16 +49,10 @@ In Gardenz, click on *Register* and add your email and password and click *Conin
 Your user account exists in Auth0 but not locally on your DB. Go to Gardenz, and click *Register*, this time click on **Log in** at the bottom and enter your email and password. There is no need to Register new emails at all, you can always sign in using your same email you used in the step above.
 You'll be redirected to Gardenz to enter additional information.
 
-#### Giving yourself Admin role
-1. Use the DB browser to update the `is_Admin` from 0 to 1.
-2. If you're on a Windows Machines with WSL, the DB Browser does not work. Fortunately, you can use sqlite3 in the terminal to access the db (CLI not GUI).
-3. In the terminal and under the Gardenz directory, type the following:
+#### Admin Role
+Ask one of the facilitators to give you the Admin role, local for gardenz or global for the entire Auth0 tenant.
```

Comment:
An alternative is to ask all of the team members to create an account on auth0.com and share the email they use with you so you can add them as a team member/collaborator. I wouldn't necessarily do this in the real-world, but because it's such a good learning experience for them, I would recommend it for multi-days. It also makes them more self-sufficient so they don't need to bug you every time they need an admin account.